lavalink/models.py

```diff
@@ -220,7 +228,26 @@ def add(self, requester: int, track: dict, index: int = None):
                 track = self.queue.pop(0)
 
         self.current = track
-        await self.node._send(op='play', guildId=self.guild_id, track=track.track, startTime=start_time)
+        options = {}
+
+        if not 0 > start_time > track.duration and start_time:
```

Comment:
I don't really like the way you use `if elif pass else`.. can you use a more elegant if-else statement ? 
Such as : 
```py
if start_time:
    if 0 > start_time > track.duration:
        raise ValueError('start_time is either less than 0 or greater than the track\'s duration')
    options['startTime'] = start_time
```